aztec-up/terraform/main.tf

```diff
@@ -71,7 +76,35 @@ resource "null_resource" "upload_public_directory" {
   }
 
   provisioner "local-exec" {
-    command = "aws s3 sync ../bin s3://${aws_s3_bucket.install_bucket.id}/"
+    interpreter = ["/bin/bash", "-c"]
+    command     = <<EOT
+      # Function to compare versions
```

Comment:
Curious when we should have commands in terraform?

Reply:
I kept them in this case as the logic is not immediately obvious.
I think this may also be pretty much the only "complicated" script that we have inside a terraform